Client/mods/deathmatch/logic/CClientExplosionManager.cpp

```diff
@@ -52,8 +69,21 @@ bool CClientExplosionManager::Hook_ExplosionCreation(CEntity* pGameExplodingEnti
 
     CClientEntity* const pResponsible = pPools->GetClientEntity(reinterpret_cast<DWORD*>(pResponsibleGameEntity->GetInterface()));
 
-    if (!pResponsible)
-        return false;
+    if (!pResponsible) {
+        if (!pGameCreator) return false;
```

Comment:
And again, for purposes of consistency with existing code, please, move return to the next line.